src/components/NewCommentForm.tsx

```diff
@@ -1,75 +1,73 @@
 import classNames from 'classnames';
-import React, { useState } from 'react';
+import React from 'react';
 import { CommentData } from '../types/Comment';
+import { useAppDispatch } from '../app/hooks';
+import { RootState } from '../app/store';
+import { useSelector } from 'react-redux';
+import {
+  AsyncPostComment,
+  setErrors,
+  clearForm,
+  setInputs,
+  FormObj,
+} from '../features/CommentForm/commentFormSlice';
+
+export const NewCommentForm: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const post = useSelector((state: RootState) => state.selecetedPost.value);
+  const errors = useSelector((state: RootState) => state.commentForm.errors);
+  const commentForm = useSelector((state: RootState) => state.commentForm);
```

Comment:
It makes no sense to keep this data in redux - you only use it here in this component. Consider remove this slice and keep data in local component state